Helper/Processor.php

```diff
@@ -33,12 +34,13 @@ public function __construct(UrlGeneratorInterface $router, TranslatorInterface $
     /**
      * Generates pagination template data
      *
-     * @param array $queryParams
-     * @param array $viewParams
+     * @param SlidingPagination $pagination
+     * @param array             $queryParams
+     * @param array             $viewParams
      *
      * @return array
      */
-    public function render($pagination, array $queryParams = array(), array $viewParams = array())
+    public function render(SlidingPagination $pagination, array $queryParams = array(), array $viewParams = array())
```

Comment:
well, passing anything else than a SlidingPagination triggers a fatal error currently. Is it really a BC break ?